src/app/server/RendezvousServer.cpp

```diff
@@ -63,6 +66,15 @@ void RendezvousServer::OnRendezvousMessageReceived(const PacketHeader & packetHe
 void RendezvousServer::OnRendezvousComplete()
 {
     ChipLogProgress(AppServer, "Device completed Rendezvous process");
+    StorablePeerConnection connection(mRendezvousSession.GetPairingSession(), mRendezvousSession.GetAdminId());
+
+    VerifyOrReturn(mStorage != nullptr);
```

Comment:
A macro of `VerifyOrReturnWithLog` may be useful to add. Without it, it seems we silently fail to do some operations.